Resources/Prototypes/Entities/Clothing/Head/hats.yml

```
@@ -761,3 +761,18 @@
     - ClothMade
     - WhitelistChameleon
     - HamsterWearable
+
+- type: entity
+  parent: ClothingHeadBase
+  id: ClothingHeadHatCentcap
+  name: Centcom Commander Cap
+  description: A grand, stylish Commander cap.
+  components:
+  - type: Sprite
+    sprite: Clothing\Head\Hats\centcomcap.rsi
+  - type: Clothing
+    sprite: Clothing\Head\Hats\centcomcap.rsi
```

```suggestion
    sprite: Clothing/Head/Hats/centcomcap.rsi
```
replace all your stuff with front slashes, NOT blackslashes

---

HUH do they really have backslashes am i crazy??